src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -172,6 +173,11 @@ const DirectoryTreeContextualMenu: React.FC<DirectoryTreeContextualMenuProps> =
                         handlePasteError(error);
                     });
                     break;
+                case ElementType.SPREADSHEET_CONFIG:
```

Comment:
You should also add the SPREADSHEET_CONFIG element type in the not clickable elements in getClickableRowStyle function in directory-content-table.tsx